newspack-theme/inc/template-tags.php

```diff
@@ -374,12 +374,20 @@ function newspack_post_thumbnail() {
 	function newspack_comment_form( $order ) {
 		if ( true === $order || strtolower( $order ) === strtolower( get_option( 'comment_order', 'asc' ) ) ) {
 
-			comment_form(
-				array(
-					'logged_in_as' => null,
-					'title_reply'  => null,
-				)
+			$comment_attributes = array(
+				'logged_in_as' => null,
+				'title_reply'  => null,
 			);
+
+			$comment_policy = get_theme_mod( 'comment_policy', '' );
+			$display_policy = get_theme_mod( 'display_comment_policy', false );
+
+			// Check if there's a comment policy set in the Customizer.
+			if ( $display_policy && '' !== $comment_policy ) {
```

Comment:
This will guard against if there's only whitespace in the Comment Policy field:

```suggestion
			if ( $display_policy && '' !== trim( $comment_policy ) ) {
```

Reply:
Oh good call! This has been added in c5829e2.